### PR TITLE
Bug 1447535 : added simple breadcrumb instead of a back button

### PR DESCRIPTION
**Pull request by iankit3**
A basic version of the breadcrumb. @sarah-clements requesting your help in understanding more about the implementation. 
What else could be a part of the breadcrumb.

https://bugzilla.mozilla.org/show_bug.cgi?id=1447535

### CODE REVIEW

File: ui/intermittent-failures/BugDetailsView.jsx [hunk 1]
```diff
@@ -139,14 +137,17 @@ const BugDetailsView = props => {
...
-                <FontAwesomeIcon
-                  icon={faArrowLeft}
-                  className="mr-1"
-                  title="go to intermittent failures view main page"
-                />
-                back
-              </Link>
+            <Breadcrumb>
+                <BreadcrumbItem>
+                  <Link title="Intermittent Failures View" to={'/'}>Treeherder</Link>
```
Comment (sarah-clements) [inline ui/intermittent-failures/BugDetailsView.jsx]: Treeherder is the parent of this app, so you don't want to have it redirect to '/' it should redirect to 'https://treeherder.mozilla.org/' here and in the other breadcrumb. The titles should also change to "Treeherder home page" or something like that.

Comment (iankit3) [reply ui/intermittent-failures/BugDetailsView.jsx]: Thanks Sarah. Made those changes.

**Comment by sarah-clements** [comment]
I tested this out locally and there needs to be a few more changes. I think the background color of the breadcrumb container would look better white instead of grey and the links to the pages that you're on need to have the 'active' prop added. It's confusing to have the 'Main view' text be clickable when you're already on the main view page (same goes for the Bug details page). You can just @ mention me here when you've made the changes instead of commenting on the bugzilla bug again :)

**Comment by iankit3** [comment]
Hi Sarah @sarah-clements, i have made the requested changes. Please have a look

File: ui/intermittent-failures/BugDetailsView.jsx [hunk 2]
```diff
@@ -137,17 +137,29 @@ const BugDetailsView = props => {
         <React.Fragment>
           <Row>
             <Col xs="12" className="text-left">
-            <Breadcrumb>
+              <Breadcrumb>
                 <BreadcrumbItem>
-                  <Link title="Intermittent Failures View" to={'/'}>Treeherder</Link>
+                  <a
+                    title="Treeherder home page"
+                    href="https://treeherder.mozilla.org/"
```
Comment (sarah-clements) [inline ui/intermittent-failures/BugDetailsView.jsx]: I think you use `/#/` here, that way if you're working locally it'll route to `localhost:5000//treeherder.mozilla.org/`.

Comment (iankit3) [reply ui/intermittent-failures/BugDetailsView.jsx]: @sarah-clements I have replaced "https://treeherder.mozilla.org/" with "/#/" for Home's href, hope that is what you mean't. Requesting your review.

**Pull request merged**